src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-best-practices-browser.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: 'OpenTelemetry JavaScript and Browser: Best practices'
+tags:
+  - Integrations
+  - Open source telemetry integrations
+  - OpenTelemetry
+metaDescription: Here are some tips for working with OpenTelemetry, browser applications, and New Relic.
+---
+
+<Callout variant="caution">
+We recommend that you do not send telemetry data from a browser application or other client-side code directly to New Relic's OTLP endpoint because this could expose your license key.
+</Callout>
+
+Support for instrumenting browser, mobile, and other client-side applications to enable real user monitoring (RUM) with OpenTelemetry is currently limited and under active development (see [OTEP-169](https://github.com/open-telemetry/oteps/issues/169)). While you may use the OpenTelemetry API or OpenTelemetry JS to instrument an application running in the browser, it is never safe to include your New Relic license key or other sensitive data as they may be easily obtained.
```

Comment:
I would remove the link to the OTEP, as it is no longer under active discussion and will likely not be the approach taken.